### ISSUE TITLE
 onMonthChange Listener calling 3 times every time i launch the app, is there any way to call only once 

### ISSUE THREAD

**Issue by kprathap23** [alamkanak/Android-Week-View#212]
@Override
    public List<WeekViewEvent> onMonthChange(int newYear, int newMonth) {

        // Populate the Month Appointments from web service for single month and added events to            //allCalendarEvents object 

         return allCalendarEvents; // declared globally 
    }

**Comment by caske33** [comment]
Duplicate of #81. See answers there.
